tests/ui/resolve/dont-compute-arg-names-for-non-fn.stderr

```diff
@@ -0,0 +1,14 @@
+error[E0412]: cannot find type `Bar` in this scope
+  --> $DIR/dont-compute-arg-names-for-non-fn.rs:6:14
+   |
+LL | impl Foo for Bar {}
+   |              ^^^
+   |
+help: you might have meant to use the associated type
+   |
+LL | impl Foo for Self::Bar {}
```

Comment:
Lol that is incorrect

Reply:
Yeah, but in the resolver we don't really have the right context to avoid this suggestion anyways.